src/query.ts

```diff
@@ -152,7 +196,7 @@ export class Query {
    *                      filter set with any value
    *
    */
-  public hasFilter(filterName: string, value?: FacetOption, filterType = 'filter'): boolean {
+  public hasFilter(filterName: string, value?: FacetOption | string | number, filterType = 'filter'): boolean {
```

Comment:
If the filter is a `Set()`, you can ask it for values with `Set.has()`.

Reply:
I'm putting my thoughts here about a small refactoring because I can't comment in the proper place and here is related to what I want to say.

I'm the author of the original implementation in v5-, I know, but today I think that I wouldn't use the `filterType` parameter. Instead, I would use a third function:

```js
hasFilter(name, value) {
 return  _whatever(this.params.filter, name, value);
}

hasExclusion(name, value) {
 return  _whatever(this.params.filter, name, value);
}
```